src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+#include <unknwn.h>
 #include "pch.h"

```

Comment:
precompiled headers should be the first header listed.

Reply:
The problem is that this is include-order sensitive.  The unknw.h has to be included before any WinRT headers, which pch.h includes, otherwise the conversion operators that are provided aren't enabled.  I could move this inside the pch.h before the appropriate headers though, would that work?

I know for certain this can't be after pch.h right now, or it won't compile.

Reply:
ah gotcha, sure moving it into the pch.h seems like a good idea. When we get the clang-format stuff set up it'll probably try to reorder these so best to fix it up ahead of that.